### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by liranr23**
Move the images to el9, bump to go 1.20.

### CODE REVIEW

**Comment by liranr23** [comment]
tests are still failing, but verified using the image.

File: pkg/must-gather-api.go [hunk 1]
```diff
@@ -8,13 +8,18 @@ import (
...
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 )
 
 var r *gin.Engine
 var db *gorm.DB
 var corsAllowedOrigins []*regexp.Regexp
 
 func main() {
+	// This line prevents controller-runtime from complaining about log.SetLogger never being called
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
```
Comment (ahadas) [inline pkg/must-gather-api.go:21]: this can be aligned with what we do elsewhere - `logging.Factory.New()`

File: go.mod [hunk 2]
```diff
@@ -1,15 +1,78 @@
 module github.com/konveyor/forklift-must-gather-api
 
-go 1.15
+go 1.20
```
Comment (ahadas) [inline go.mod:3]: so here we say that the minimum is 1.20 and then compile it with 1.19? :)

Comment (liranr23) [reply go.mod:3]: well.. it worked locally creating the image... pretty strange indeed. i will switch to 1.19

File: pkg/must-gather-api.go (hunk 1 above)
Comment (ahadas) [reply pkg/must-gather-api.go:21]: ah I see, logging is an internal package..
ok, so why setting dev-mode to true?

**Review by ahadas — APPROVED** [review]

**Pull request merged**